### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by jloures**

### CODE REVIEW

**Comment by jloures** [comment]
Created a pull request. Will look into it on Sunday

**Comment by jloures** [comment]
Looks really good. Actually way better than I expected. The algorithm is definitely NOT machine learning but it works for this milestone (in contrast with jonathans)

File: Application.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            sumB = 0     
+            for k in range (0, 6):
+                sumA += dic[matrixData[x, k]]
+                sumB += dic[matrixData[x, k+6]]
+                
+            avgA = sumA / 6
+            avgB = sumB / 6   
+            scoreA = matrixData[x, -2]
+            scoreB = matrixData[x, -1]      
+            x_adjustment = (((scoreA - scoreB) / 2 - (avgA - avgB)) / 2) * 11
```
Comment (jloures) [inline Application.py]: why 11? Shouldn't this be 12?

**Comment by jloures** [comment]
Once you explain to me the reason for 11, you are good to merge
👍

File: Application.py (hunk 1 above)
Comment (vivienfan) [reply Application.py]: theoretically this should multiply by just 2, recall: score_diff = int( avg_diff \* 2 )
and this x_adjustment is to adjust the rating for each player, not the entire team; hence, should not multiply by 12
the purpose of 11 is to enlarger x_adjustment, as a reverse of int() function. And I figure that here we should use a prime number to do the job; and by trail and error, i found 11 is the best fit.
does this answer your question?
FYI, the minimal game instances to get the correct ratings for this case is 30

**Pull request merged**